### PR TITLE
Update README.md with submission guidelines

### PR DESCRIPTION
**Pull request by captainsafia**
I think this is a useful thing to have.

Let me know if it is not necessary or if tweaks should be made.

cc/ @ellisonbg @rgbkrk

### CODE REVIEW

**Comment by rgbkrk** [comment]
That's looking pretty good to me.

Did you mean to submit this against your own fork?

File: README.md [hunk 1]
```diff
@@ -7,3 +7,26 @@ This the place to suggest large enhancements that may impact several repositorie
 * Additions/changes to the message spec
 * API design that is consumed and produced in several contexts (e.g. the notebook kernel API)
 * The notebook format
+
+## Enhancement Proposal Submission Guidlines
+
+When submitting a enhancement proposal, please include the following information in your submission.
```
Comment (rgbkrk) [inline README.md:13]: _an enhancement_

File: README.md [hunk 2]
```diff
@@ -7,3 +7,26 @@ This the place to suggest large enhancements that may impact several repositorie
...
+
+1. The problem that this enhancement addresses. If possible include code or anecodotes to describe this problem to readers.
+2. A brief (1-2 sentences) overview of the enhancement you are prposing. If possible include hypothetical code sample to describe how the solution would work to readers.
```
Comment (rgbkrk) [inline README.md:16]: _are proposing_

**Comment by captainsafia** [comment]
@rgbkrk — Nope. I thought the "Create Fork and Submit Pull Request" button in the file-editing interface of Github would submit to upstream, but apparently not. 

I just sent another PR into the main repo.

File: README.md (hunk 2 above)
Comment (captainsafia) [inline README.md:16]: I KNOW HOW TO TYPE I SWEAR I DON'T HAVE FAT FINGERS. 

I'll fix these.

giphy-2

Comment (rgbkrk) [inline README.md:16]: Hahahah it's ok, I read it before nit picking.

Totally typing this from a phone on the road.

**Comment by rgbkrk** [comment]
Awesome!

**Comment by captainsafia** [comment]
@rgbkrk — typos fixed. PR in main repo.

**Comment by rgbkrk** [comment]
Your cat GIFs are so great.

**Pull request merged**